modules/alert/README.md

```diff
@@ -59,21 +59,38 @@ No modules.
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | **display_name** | A short name or phrase used to identify the condition in dashboards, notifications, and incidents. To avoid confusion, don't use the same display name for multiple conditions in the same policy. | string | n/a | **yes** |
-| **condition_threshold** | Object containing configuration for the condition threshold/s  | object(any) | n/a | **yes** |
+| **condition_threshold** | Object containing configuration for the condition threshold/s  | object(any) | n/a | no |
+| **condition_monitoring_query_language** | Object containing configuration for the Monitoring Query Language (MQL) query. | object(any) | n/a | no |
+| **condition_matched_log** | Object containing configuration for a condition that checks for log messages matching given constraints. | object(any) | n/a | no |
 
 ### **policies.conditions.condition_threshold** supported attributes
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| **comparison** | The comparison to apply between the time series (indicated by filter and aggregation) and the threshold (indicated by threshold_value). The comparison is applied on each time series, with the time series on the left-hand side and the threshold on the right-hand side. Only COMPARISON_LT and COMPARISON_GT are supported currently. Possible values are COMPARISON_GT, COMPARISON_GE, COMPARISON_LT, COMPARISON_LE, COMPARISON_EQ, and COMPARISON_NE. | string |  | **yes** |
+| **comparison** | The comparison to apply between the time series (indicated by filter and aggregation) and the threshold (indicated by threshold_value). The comparison is applied on each time series, with the time series on the left-hand side and the threshold on the right-hand side. Only COMPARISON_LT and COMPARISON_GT are supported currently. Possible values are COMPARISON_GT, COMPARISON_GE, COMPARISON_LT, COMPARISON_LE, COMPARISON_EQ, and COMPARISON_NE. | string | n/a | **yes** |
```

Comment:
add the default value `COMPARISON_GT` , and change it to not required since it will the default